wp/sections/feature.tex

```diff
@@ -0,0 +1,67 @@
+\par As described above, diKTat is very configurable and comfortable. But these are not all of its advantages and features. Below will be presented and described unusual and important killer-feature diKTat.
+
+\subsection{Configuration file}
+\par
+It's worth starting with the configuration files. This is a file in which the user can manually turn rules on and off or configure the rules settings. Below is one of the rules in the configuration file.
+\begin{center}
+- name: FILE\underline{ }IS\underline{ }TOO\underline{ }LONG\\
+	enabled: true\\
+	configuration:\\
+	maxSize: '2000'\\
+	ignoreFolders: ' '\\
+\end{center}
+Each rule in this file consists of 3 parts: name - the name of the rule, enabled - the ability to enable or disable the rule, configuration - parameters for the rules. With the first two, everything is obvious. The third parameter is less obvious. The configuration is passed "properties" to check in this rule. For example, if you have a rule "FILE\underline{ }IS\underline{ }TOO\underline{ }LONG" - checking the number of lines in a Kotlin file, then the user can configure the maximum number of lines in the file - by changing the "maxSize" in the configuration,or the user can specify paths to folders that do not need to be checked - by writing the path before these daddy in "ignoreFolders". \\
```

Comment:
Write about the functionality that Petr added - that properties is validated and promts you the nearest option in case you have made a missprint/typo